cmd/virt-handler/virt-handler.go

```diff
@@ -130,6 +131,11 @@ func (app *virtHandlerApp) Run() {
 		cache.Indexers{},
 	)
 
+	pvcSharedInformer := cache.NewSharedIndexInformer(
```

Comment:
Can you revert that? We need a different solution for virt-handler. That would effectively mean that we have `N` watchers which will watch all PVCs for `N` nodes.

Reply:
done